src/test/scala-2.11/markets/participants/TestOrderCanceler.scala

```diff
@@ -30,7 +30,7 @@ import scala.concurrent.ExecutionContext.Implicits.global
 
 case class TestOrderCanceler(initialDelay: FiniteDuration,
                              markets: mutable.Map[Tradable, ActorRef],
-                             tickers: mutable.Map[Tradable, Agent[immutable.Seq[Tick]]])
+                             tickers: mutable.Map[Tradable, Agent[Tick]])
   extends OrderCanceler[TestOrderCancellationStrategy] {
 
   val outstandingOrders = mutable.Set.empty[Order]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2168327552/issues/source?bid=2921361&fileBranchId=3162043#l36)